### ISSUE TITLE
Missing `-f` option in 1.2

### ISSUE THREAD

**Issue by jakul** [DavidVujic/poetry-multiproject-plugin#28]
**Describe the bug**
The `-f` option has been removed in v1.2

**To Reproduce**
* Install 1.2

**Error Log**

```shell
root@7a84c5cfeaad:/home/presets/echo# poetry self add poetry-multiproject-plugin==1.1.6

Updating dependencies
Resolving dependencies... (0.6s)

Writing lock file

Package operations: 4 installs, 0 updates, 0 removals

  • Installing mypy-extensions (1.0.0)
  • Installing typing-extensions (4.5.0)
  • Installing mypy (0.991)
  • Installing poetry-multiproject-plugin (1.1.6)
root@7a84c5cfeaad:/home/presets/echo# poetry build-project --help

Description:
  Builds a package, as a tarball and a wheel by default.

Usage:
  build-project [options]

Options:
  -f, --format=FORMAT        Limit the format to either sdist or wheel.
  -h, --help                 Display help for the given command. When no command is given display help for the list command.
  -q, --quiet                Do not output any message.
  -V, --version              Display this application version.
      --ansi                 Force ANSI output.
      --no-ansi              Disable ANSI output.
  -n, --no-interaction       Do not ask any interactive question.
      --no-plugins           Disables plugins.
      --no-cache             Disables Poetry source caches.
  -C, --directory=DIRECTORY  The working directory for the Poetry command (defaults to the current working directory).
  -v|vv|vvv, --verbose       Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug.
root@7a84c5cfeaad:/home/presets/echo# poetry self add poetry-multiproject-plugin~=1.1

Updating dependencies
Resolving dependencies... (0.3s)

Writing lock file

Package operations: 0 installs, 1 update, 0 removals

  • Updating poetry-multiproject-plugin (1.1.6 -> 1.2.0)
root@7a84c5cfeaad:/home/presets/echo# poetry build-project --help

Description:
  Builds a package, as a tarball and a wheel by default.

Usage:
  build-project [options]

Options:
      --with-top-namespace=WITH-TOP-NAMESPACE  To arrange relative includes, and to modify import statements.
  -h, --help                                   Display help for the given command. When no command is given display help for the list command.
  -q, --quiet                                  Do not output any message.
  -V, --version                                Display this application version.
      --ansi                                   Force ANSI output.
      --no-ansi                                Disable ANSI output.
  -n, --no-interaction                         Do not ask any interactive question.
      --no-plugins                             Disables plugins.
      --no-cache                               Disables Poetry source caches.
  -C, --directory=DIRECTORY                    The working directory for the Poetry command (defaults to the current working directory).
  -v|vv|vvv, --verbose                         Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug.
root@7a84c5cfeaad:/home/presets/echo# 
```

**Expected behavior**
Able to use `-f sdist` to specify that I only want an sdist to be built.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Linux
 - Python version:  3.6.9
 - Poetry version: 1.3.2

**Additional context**
I think that adding the `--with-top-namespace` option has replaced the `-f` option instead of adding a second option.

**Comment by DavidVujic** [comment]
Thank you for reporting! Yes, it seems I have messed something up with the latest release. I'm having a look at it right now.

**Comment by DavidVujic** [comment]
I believe this is now fixed and a new version is available to install: v1.2.1. Huge thanks for reporting this! ⭐

**Comment by jakul** [comment]
Hey David,

Thanks for the speedy action!

I've tested the new version and it fixes the issue for me, thank you!

-- Craig Blaszczyk
